### PR TITLE
Improve small screen styling

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Load the GUI
- Shrink your browser down to a phone size
- Use it and see that its useable again
- Deploy the GUI to a cloud
- Load the IP up on your phone
- Further check

_The help dropdown. I couldn't resolve quickly. So moved out of scope for this branch_

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6968/

**Comment by kadams54** [comment]
Model actions look great. The rest of the header (breadcrumb, model switcher, help, and search)? Not so much. Whether this branch passes QA kind of depends on your definition of "useable". I can't switch to other models or search the charmstore, but the parts impacted by the actual code changes do look better.

**Comment by anthonydillon** [comment]
@kadams54 thanks, I will work on the header and model switcher.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6974/

**Comment by kadams54** [comment]
Ah, much nicer, thanks!

File: jujugui/static/gui/src/app/components/model-actions/_model-actions.scss [hunk 1]
```diff
@@ -1,6 +1,11 @@
 // scss-lint:disable IdSelector
 #model-actions-container { // sass-lint:disable-line no-ids, id-name-format
-  @extend .vertical-center;
+  @media only screen and (min-width: 600px) {
```
Comment (kadams54) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: Out of curiousity, why is this here and not in `_mobile.scss`?

Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: The mobile file contains styles that take effect at < 600px. This styling uses min-width and therefore only applies when > 600px. I didn't want to complicate the mobile file by adding desktop styles.

Comment (kadams54) [inline jujugui/static/gui/src/app/components/model-actions/_model-actions.scss]: Got it, thanks.

**Comment by Lukewh** [comment]
One small thing - can you get rid of the left margin on `.env-switcher__toggle` so that the username and model switcher are inline? Otherwise QA OK

**Comment by Lukewh** [comment]
QA OK. If test pass, ship it 👍

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6992/

**Comment by anthonydillon** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge
